# Hekateros hekateros_control/SetRobotMode service

industrial_msgs/RobotMode mode    # robot mode: AUTO MANUAL
---
int8 rc                           # return code
